### PR TITLE
chore: Improve search experience

### LINKED ISSUE #1122: [Gallery][iOS] AutoSuggestBox- The indentation for all the displayed control names with definitions is incorrect.
## Current behavior
autosuggest issue 1
autosuggest issue 2

## Expected behavior
Screenshot_20240618_225624_Uno Gallery

## How to reproduce it (as minimally and precisely as possible)

1. Open Uno Gallery APP
2. Tap "Autosuggest Box" from the "UI Components" list.
3. Type any letter in the ' Autosuggest box with searchbox experience'
4. Make sure combo box list opens with all the controls names.
5. Notice that the indentation for all the displayed control names with definitions is incorrect.

## Environment

Nuget Package:

Package Version(s):

Affected platform(s):

- [X] iOS
- [ ] Android
- [ ] WebAssembly
- [ ] WebAssembly renders for Xamarin.Forms
- [ ] Windows
- [ ] Build tasks

Visual Studio:

- [ ] 2017 (version: )
- [ ] 2019 (version: )
- [ ] for Mac (version: )

Relevant plugins:

- [ ] Resharper (version: )

### PR DESCRIPTION
**Pull request by morning4coffe-dev**
GitHub Issue (If applicable): closes #1122

## PR Type
What kind of change does this PR introduce?
- Bugfix

## What is the new behavior?
fix search result visuals

## PR Checklist
Please check if your PR fulfills the following requirements:
- [ ] Tested on iOS.
- [x] Tested on Wasm.
- [x] Tested on Android.
- [x] Tested on UWP.
- [x] Tested in both **Light** and **Dark** themes.
- [x] Associated with an issue (GitHub or internal)

## Other information
part of https://github.com/unoplatform/Uno.Gallery/issues/1122

### CODE REVIEW

**Review by MartinZikmund — APPROVED** [review]

**Review by Xiaoy312 — APPROVED** [review]

**Comment by MartinZikmund** [comment]
@agneszitte can you merge this?

**Comment by Xiaoy312** [comment]
the ui tests have been repeatedly failing, for 4 reruns
can you verify locally that everything is still fine?

**Comment by morning4coffe-dev** [comment]
@Xiaoy312 I think this issue with the CI is coming from master, but I think we discussed with @agneszitte last week on different issue with the same error, it should be ok to merge. I am unfortunately not able to test on iOS / macOS.

File: Uno.Gallery/Uno.Gallery.Shared/Views/Shell.xaml.cs [hunk 1]
```diff
@@ -213,34 +214,45 @@ private void NavigationViewControl_SizeChanged(object sender, SizeChangedEventAr
...
+				sender.ItemsSource = filteredSamples;
+			}
 		}
 
 		private void SamplesSearchBox_SuggestionChosen(AutoSuggestBox sender, AutoSuggestBoxSuggestionChosenEventArgs args)
 		{
-			(Application.Current as App)?.SearchShellNavigateTo(args.SelectedItem as Sample);
+			var sample = args.SelectedItem as Sample;
+
+			if (sample.Title.Contains("No suggestions found"))
```
Comment (carldebilly) [inline Uno.Gallery/Uno.Gallery.Shared/Views/Shell.xaml.cs:250]: ...

Can we at least use a constant?

Comment (morning4coffe-dev) [reply Uno.Gallery/Uno.Gallery.Shared/Views/Shell.xaml.cs:250]: Good suggestion, @carldebilly, thanks. It's not the best, but it works better then before 😅

**Review by agneszitte — APPROVED** [review]

**Pull request merged**